arrows/ocv/register_algorithms.cxx

```diff
@@ -37,6 +37,7 @@
 #include <arrows/ocv/feature_detect_extract_ORB.h>
 #include <arrows/ocv/feature_detect_extract_SIFT.h>
 #include <arrows/ocv/feature_detect_extract_SURF.h>
+#include <arrows/ocv/inpaint.h>
 #include <arrows/ocv/image_io.h>
```

Comment:
Pedantic: `'n'` > `'m'`:
```suggestion
#include <arrows/ocv/image_io.h>
#include <arrows/ocv/inpaint.h>
```

(I wonder if the registrations should be in some sort of order, also? Right now it seems they aren't.)

Reply:
It seems like they're loosely grouped by functionality but yes, I agree this isn't great.

Reply:
I wouldn't be surprised if they're in the order the algorithms were created 🙂. The second comment stems mainly from noting that `inpaint` and `image_io` were in that order for registration, also, but then followed by something that was definitely *not* lexicographical order.